assets/CSS/style.css

```diff
@@ -0,0 +1,230 @@
+* {
+  box-sizing: border-box;
+}
+.body {
+  font-size: 100%;
+}
+.main-container {
+  width:100%;
+  height: 4000px;
+  font-family: 'Lato', 'Oswald', 'Ubuntu', 'Roboto';
+}
+.grid-header {
+  background-color: #8c8c8c;
+  width:100%;
+  height: 2.5%;
+  display: grid;
+  grid-template-columns: 20% 13% 13% 13% 13% 13% auto;
+  grid-template-rows: 100%;
+}
+.header-grid-item {
+  text-align: center;
+}
+.header-grid-item p {
+  text-align: center;
+  height: 100%;
+  display: inline-grid;
+  align-content: center;
+  color:#fff;
+  font-size: 1.5em;
+  font-weight: bold;
+}
+.latest-articles-title {
+  font-size: 1.5em;
+  font-weight: bold;
+  display: block;
+  position: absolute;
+  top: 150px;
+  left: 45%;
+  text-align: center;
```

Comment:
This line appears 10 times in the stylesheet, try using a separate class selector for repeating classes like these.

Reply:
`display: grid` is also a good candidate for same. Of course, you are free to disagree with me on this, simply comment here :)

Reply:
The "text-align" property has been put in a common class (class="center")
display: grid is also better for us we changed it :+1: :+1:  
Don't Miss... part has been centered (We hope so :D)
some classes which have the same properties (more than one) have been combined together in CSS

And also thank you for your great review Maya :1st_place_medal: 
We are gonna send another review from the dashboard hope to see you :wave:

Reply:
I am glad you liked my review! 😄 . I Like Y'all project. Checking out changes now.